examples/jest/README.md

```diff
@@ -42,3 +42,10 @@ nested:
     0.00392 ms (+/- 0.00068 ms) from 100 iterations
 - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 ```
+
+## Environment Variables
+
+| Name                        | Option | Description                                                                                |
+| :-------------------------- | :----- | :----------------------------------------------------------------------------------------- |
+| KELONIO_KEEP_STATE_AT_START | FALSE  | Generates a fresh ".kelonio.state.json" each time jest is executed                         |
```

Comment:
Can you please document in this readme what the default behaviour is?

Reply:
Good point, I will address this - thank you

Reply:
Addressed in a new commit

Reply:
According to what you have in the code, this needs to be "true" lower case to work. So "false" should probably also be lowercase. But I would actually just leave the "false" option out because that's the default.